js/forum/src/components/EditUserModal.js

```diff
@@ -37,22 +38,32 @@ export default class EditUserModal extends Modal {
       <div className="Modal-body">
         <div className="Form">
           <div className="Form-group">
-            <label>Username</label>
+            <label>{app.translator.trans('core.forum.edit_user.username_heading')}</label>
             <input className="FormControl" placeholder={extractText(app.translator.trans('core.forum.edit_user.username_label'))}
               bidi={this.username} />
           </div>
 
           {app.session.user !== this.props.user ? [
             <div className="Form-group">
-              <label>Email</label>
+              <label>{app.translator.trans('core.forum.edit_user.email_heading')}</label>
               <div>
                 <input className="FormControl" placeholder={extractText(app.translator.trans('core.forum.edit_user.email_label'))}
                   bidi={this.email} />
               </div>
+              {!this.isActivated() ? (
+                <div>
+                  {Button.component({
+                    className: 'Button Button--block',
+                    children: app.translator.trans('core.forum.edit_user.activate_button'),
+                    loading: this.loading,
+                    onclick: this.activate.bind(this)
+                  })}
+                </div>
+              ) : ''}
```

Comment:
What I meant with "double negation" is that there is still an `else` clause at the end.

And when I see something like this:
~~~js
if ( ! something) {
  // Do A
} else {
  // Do B
}
~~~

I automatically think: Why not turn it around?

~~~js
if (something) {
  // Do B
} else {
  // Do A
}
~~~

That way, there will be less negation in my head when trying to understand the if and else blocks.

However, we're only using the ternary operator here because there is no inline if statement - therefore I agree with you that it's better to have the one condition that we are checking for here come first - the `else` here is basically a no-op.